airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -38,7 +39,14 @@ def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
     def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
         list_id = self._get_list_id(response.url)
         for user in response.iter_lines():
-            yield {"email": user.decode(), "listId": list_id}
+            user_email = user.decode()
```

Comment:
**Preferred option**
Can we instead use this endpoint for events? https://api.iterable.com/api/docs#events_User_events_by_userId

**Less preferred option**
instead of skipping it in here, can we instead skip it in child streams? this way we don't lose data in the ListUsers stream but other streams which depend on it don't fail. It's fine in this case to skip IMO, unless we know for sure there is a way to obtain this data.

Reply:
Sadly, preferred option won't work for all the cases for us, see https://github.com/airbytehq/oncall/issues/43#issuecomment-984546420

Less preferred option is ok, working on it.